openbox/resist.c

```diff
@@ -59,22 +59,20 @@ static gboolean resist_move_window(Rect window,
        window edge available, without going all the way from
        bottom-to-top in the stacking list
     */
-    if (!snapx) {
-        if (ct < tb && cb > tt) {
-            if (cl >= tr && l < tr && l >= tr - resist)
-                *x = tr, snapx = TRUE;
-            else if (cr <= tl && r > tl &&
-                     r <= tl + resist)
-                *x = tl - w + 1, snapx = TRUE;
-            if (snapx) {
-                /* try to corner snap to the window */
-                if (ct > tt && t <= tt &&
-                    t > tt - resist)
-                    *y = tt + 1, snapy = TRUE;
-                else if (cb < tb && b >= tb &&
-                         b < tb + resist)
-                    *y = tb - h, snapy = TRUE;
-            }
+    if (ct < tb && cb > tt) {
+        if (cl >= tr && l < tr && l >= tr - resist)
+            *x = tr, snapx = TRUE;
+        else if (cr <= tl && r > tl &&
```

Comment:
do all of these ifs still need to line-wrap for 80 columns?

Reply:
Now that I double checked, the line-wraps are not necessary.